lib/ace/virtual_renderer.js

```diff
@@ -428,18 +427,21 @@ var VirtualRenderer = function(container, theme) {
     };
 
     this.onGutterResize = function() {
-        var gutterWidth = this.$showGutter ? this.$gutter.offsetWidth : 0;
-        if (gutterWidth != this.gutterWidth)
-            this.$changes |= this.$updateCachedSize(true, gutterWidth, this.$size.width, this.$size.height);
-
-        if (this.session.getUseWrapMode() && this.adjustWrapLimit()) {
-            this.$loop.schedule(this.CHANGE_FULL);
-        } else if (this.$size.$dirty) {
-            this.$loop.schedule(this.CHANGE_FULL);
-        } else {
-            this.$computeLayerConfig();
-            this.$loop.schedule(this.CHANGE_MARKER);
-        }
+        var that = this;
+        setTimeout(function() {
```

Comment:
Why is this needed, if it is to not call gutter.offsetWidth, i think we can safely remove it https://github.com/ajaxorg/ace/commit/e701c27e30ddf47365dcaee9f67c3a2d23f2bb92#diff-1d3adf2538ef07e28972236f3a26311aR431

Reply:
It was needed because it would render the change in the same frame causing the frame to take too long to render. With this change the update is applied in the the following frame.

Reply:
This works well while scrolling, and another render is going to happen anyway, but when pressing enter it render same thing twice, and there is small visible jump.